CHANGELOG.asciidoc

```diff
@@ -27,6 +27,7 @@ image::https://raw.githubusercontent.com/apache/tinkerpop/master/docs/static/ima
 * Deprecated `TraversalStrategies.applyStrategies()`.
 * Deprecated Jython support in `gremlin-python`.
 * Reverted: Modified Java driver to use IP address rather than hostname to create connections.
+* TINKERPOP-2320 Allow custom XMLInputFactory to be used with GraphMLReader.
```

Comment:
No need to add the JIRA number - those get added separately on release.

Reply:
OK, removed it. Thanks!